MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -522,6 +527,10 @@ public ReadOnlySpan<byte> Invoke(ushort ordinal, bool offsetsOnly = false)
                     return uidxrf;
                 case 600:
                     return tjoinrou;
+                case 861:
```

Comment:
this may answer my prior question - if we're exporting variables then it makes sense